Python/libraries/recognizers-date-time/recognizers_date_time/date_time/base_datetimeperiod.py

```diff
@@ -221,9 +221,9 @@ def extract(self, source: str, reference: datetime = None) -> List[ExtractResult
         date_ers = self.config.single_date_extractor.extract(source, reference)
         time_ers = self.config.single_time_extractor.extract(source, reference)
 
-        tokens: List[Token] = list()
+        tokens = []
         tokens.extend(self.match_simple_cases(source, reference))
-        tokens.extend(self.merge_two_time_points(source, reference, date_ers, time_ers))
+        tokens.extend(self.merge_two_time_points(source, reference, list(date_ers), list(time_ers)))
```

Comment:
What do you think about changing `list()` to `[]`?
```suggestion
        tokens.extend(self.merge_two_time_points(source, reference, [date_ers)], [time_ers]))
```